app/pages/BuyNBTC/BuyNBTC.tsx

```diff
@@ -11,7 +12,8 @@ import { Tabs } from "~/components/ui/tabs";
 import { ArrowUpRight } from "lucide-react";
 
 export function BuyNBTC() {
-	const { balance: nBTCBalance } = useNBTCBalance();
+	// TODO: it doesn't get automatically refresehed
```

Comment:
we need to solve this refresh. Previously it was automatically refreshed, because we were using nested query client, and that caused page reload on the refresh of balances in useNBTC / useBuySellNBTC